This patch applies from WRKSRC_SUBDIR, ie. from cpp/ , but even
when building in that subdir, the tools/ directory is built
as well. That is why the patch applies to ..

- use system abseil instead of fetching it on our own.
- force C++17 because abseil has it, too.

--- ../tools/cpp/CMakeLists.txt.orig	2022-01-11 10:41:37.000000000 +0100
+++ ../tools/cpp/CMakeLists.txt	2022-01-22 17:06:39.284013000 +0100
@@ -18,7 +18,7 @@
 
 # Pick the C++ standard to compile with.
 # Abseil currently supports C++11, C++14, and C++17.
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 project (generate_geocoding_data)
